devops/jobs/CreateDataCzar.groovy

```diff
@@ -42,11 +42,11 @@ class CreateDataCzar{
                 stringParam('ORGANIZATION',
                         'Name of organization to create data czar. e.g alaskax')
                 booleanParam('CREATE_ORG', true,
-                        'true if create new organization or false if organization is already created.')
+                        'Check if create new organization or Uncheck if organization is already created.')
                 stringParam('USER_EMAIL',
                         'User Email address to generate Data Czar')
                 fileParam('user_gpg_key.gpg', 
-                        'Paste the GPG key for encryption of aws credentials')
+                        'Attach the GPG key file for encryption of aws credentials')
```

Comment:
Should this be a file or should we just paste in the public key?  Also could you add the word public to this description?

Reply:
We should attach the file for public key, this is same as previous workflow.
I am adding public in this message.